dashboard/app/helpers/levels_helper.rb

```diff
@@ -787,7 +787,18 @@ def redirect_under_13_without_tos_teacher(level)
     return false unless level.game == Game.applab || level.game == Game.gamelab || level.game == Game.weblab
 
     if current_user && current_user.under_13? && current_user.terms_version.nil?
-      error_message = current_user.teachers.any? ? I18n.t("errors.messages.teacher_must_accept_terms") : I18n.t("errors.messages.too_young")
+      if current_user.teachers.any?
+        error_message = I18n.t("errors.messages.teacher_must_accept_terms")
+      else
+        error_message = I18n.t("errors.messages.too_young")
+        FirehoseClient.instance.put_record(
```

Comment:
Should this also log a user_id or something so we can de-dupe in our SQL queries when trying to count how many students are denied?

Reply:
Good call. Done.